### PR TITLE
Add support for additional keypad types

### PR DESCRIPTION
**Pull request by koreth**
This adds support for Hybrid Seetouch and Tabletop Seetouch keypads.

### CODE REVIEW

File: pylutron/__init__.py [hunk 1]
```diff
@@ -184,7 +184,10 @@ def _parse_area(self, area_xml):
       for device_xml in devs:
         if device_xml.tag != 'Device':
           continue
-        if device_xml.get('DeviceType') in ('SEETOUCH_KEYPAD', 'PICO_KEYPAD'):
+        if device_xml.get('DeviceType') in (
+                'SEETOUCH_KEYPAD',
```
Comment (thecynic) [inline pylutron/__init__.py]: 4 space indent below the if?

File: pylutron/__init__.py [hunk 2]
```diff
@@ -222,13 +225,20 @@ def _parse_button(self, component_xml):
     """Parses a button device that part of a keypad."""
     button_xml = component_xml.find('Button')
     name = button_xml.get('Engraving')
+    button_type = button_xml.get('ButtonType')
+    direction = button_xml.get('Direction')
+
+    # Hybrid keypads have dimmer buttons which have no engravings.
+    if button_type == 'SingleSceneRaiseLower':
+      name = 'Dimmer ' + direction
+
```
Comment (thecynic) [inline pylutron/__init__.py]: remove newline.

File: pylutron/__init__.py [hunk 3]
```diff
@@ -222,13 +225,20 @@ def _parse_button(self, component_xml):
     """Parses a button device that part of a keypad."""
     button_xml = component_xml.find('Button')
     name = button_xml.get('Engraving')
```
Comment (thecynic) [inline pylutron/__init__.py]: will this just be missing in the db completely? or is it an empty string? Sorry, I don't have one of these :(

Comment (koreth) [reply pylutron/__init__.py]: It's missing entirely. Example from my configuration:

```xml
<Button Name="Button 18" UUID="2471" ButtonType="SingleSceneRaiseLower" PrimaryActionType="3" SceneSaver="false" CycleDim="false" AllowDoubleTap="true" StopIfMoving="false" Direction="Lower" ProgrammingModelID="3762" LedLogic="4" ReverseLedLogic="false">
```

**Comment by koreth** [comment]
I added one other keypad type to this PR (for Lutron's battery-powered tabletop keypad). Seemed reasonable to bundle it in this PR but can do a separate one if you prefer.

**Comment by bdunks** [comment]
Would it be possible to go ahead and merge in these changes?  I think the outstanding review may have been addressed in a comment.

I'd to add support for an additional keypad type, `'HOMEOWNER_KEYPAD'`, which is the name the repeater uses for keypads created from within the Lutron Connect app, which don't have a physical counterpart.

It would conflict with these changes.

Thanks!

**Pull request merged**